### ISSUE TITLE
Alternative to TigerVNC?

### ISSUE THREAD

**Issue by TechyAditya** [tuanpham-dev/termux-ubuntu#25]
VNC can't play YouTube videos reliably. It has heavy frame drops. It's simply reliable. I've checked sites like anydesk, they don't have support for arm64. Any other programs that can share virtual display?

**Comment by tuanpham-dev** [comment]
@TechyAditya Try this https://docs.andronix.app/xsdl/xsdl-basics
However I think you better play youtube in android app with PiP enabled.

**Comment by TechyAditya** [comment]
Well, the app is client-sided. But the problem is server-sided, it still uses vnc server. I tried xrdp, but it still is a bit laggy on videos even if it's more reliable than vnc client. Seems like it's the issue with hardware acceleration. Apps like Anydesk aren't built for arm64 yet. 

I also tried running zoom web app in chromium, but there's a black screen on video boxes and it is audio-only. Camera and mic access is still not there. Opening chromium and firefox at the same time totally slows down the phone and crashes android apps. 

I use laptop or android TV to access Linux on this mobile under the same wifi mode (sometimes mobile hotspot too), hence want to play youtube videos on Linux (even if it's not necessary). One thing to agree is that this project on Snapdragon 720G phone is far better than my old 2015 laptop running ubuntu.
